src/app/app.component.ts

```diff
@@ -8,12 +13,11 @@ import {TestService} from './test.service';
 })
 export class AppComponent implements OnInit {
   title = 'samplecardexample';
+  sampleData = this.store.pipe(select(itemData));
 
-  public sampleData:any[]=[];
-  constructor(private testService: TestService) { }
+  constructor(private store: Store<{item:Item[]}>) { }
```

Comment:
private readonly store: Store<{item:Item[]}>

please use explicit public, private or protected

Reply:
private variables should include _ at the start, e.g. _store

Reply:
updated